content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md

```diff
@@ -0,0 +1,153 @@
+
+Model performance can be obtained by comparing various runs. In the image below, all six runs have been selected and we see that the model with epochs=20, performs best with the least loss and mean absolute error (MAE).
+
+![](/img/fig-3-multiple-runs-compared.png "Figure 5. Comparison of all runs based on a relevant metric. ")
+
+Now that the best model has been identified, it can be registered programmatically or through the MLflow Web UI. Once registered, the model can be deployed to the appropriate staging-production region. 
+
+![](/img/fig-4-model-registry-different-models.png "Figure 6. Model registry with different models. ")
+
+**Code Repository**
+
+A code repository is a storage location for code and other software development assets, such as documentation, tests, and scripts. They are often used to manage and organize a software project's codebase and collaborate with other project developers.
+
+The source code for the model is stored in a private repository on GitHub then seamlessly pulled into the IDE using git commands. The updated model can be pushed to different branches of the source repo. 
+
+![](/img/code-repository.png "Figure 7.  Source code for models is stored on Github in a private repository. ")
+
+**Model Deployment**
+
+The next step is to deploy the best model chosen from the MLflow experiment. Here we are using KServe which is a standard Model Inference Platform on Kubernetes, built for highly scalable use cases.
+
+Kube Flow Serving, also known as KServe, is a component of the Kube Flow ecosystem and is designed to serve machine learning models in Kubernetes environments. Kube Flow itself is an open-source platform for deploying, monitoring, and managing machine learning models on Kubernetes.
+
+KServe provides several features and benefits:
+
+1. Scalability: It is designed to handle serving machine learning models at scale. You can easily scale up or down based on the traffic and load requirements.
+2. Multi-Framework Support: KServe supports serving models trained in various machine learning frameworks, such as Tensor Flow, PyTorch, scikit-learn, and others.
+3. Advanced Deployment Strategies: KServe supports various deployment strategies, for example  Canary deployments or Blue-Green deployments,  allowing you to roll out new model versions gradually and monitor their performance.
+4. Monitoring and Metrics: It provides metrics and monitoring capabilities, allowing you to keep track of model performance, errors, and other relevant statistics.
+5. Customization: KServe is highly customizable, allowing you to define how you want to serve your models, handle preprocessing and post-processing tasks, and even add custom logic to your serving containers.
+6. Integration with Kube Flow Pipelines: If you're using Kube Flow Pipelines for end-to-end machine learning workflows, KServe can be integrated seamlessly to deploy models as part of your pipeline.
```

Comment:
I think Denis had a question as to whether Kube Flow was one word (KubeFlow) or not